### ISSUE TITLE
Image Broken:"VPR/query_images/query_images_2/@120.43912@36.59905333333333@.png"

### ISSUE THREAD

**Issue by Frost24K** [hmf21/AerialVL#4]
Hi,

I successfully downloaded the VPR file and unzipped it, however, when I evaluated the VPR metric, I encountered an issue with the file `query_images/query_images_2/@120.43912@36.59905333333333@.png`. It failed to load correctly, resulting in the error "SyntaxError: broken PNG file (chunk b'\x93\xed\xee\xf6')".

Thank you for looking into this.

**Comment by hmf21** [comment]
Hi @Frost24K,
This file in my computer can be  loaded successfully.
I think maybe there is a problem with decompression, and the correct picture is supplied below.
Please contact me if there are any other problems.
![@120 43912@36 59905333333333@](https://github.com/user-attachments/assets/d5b2284a-490c-4a6b-9a1f-631e42de34dd)

**Comment by Frost24K** [comment]
Hi, 
Thank you for helping me resolve this issue.
Best

**Comment by Frost24K** [comment]
Hi,
Big pardon, I noticed a discrepancy when loading the VPR_test dataset. Specifically, the total number of drone images I found was 18,359 (query_images_1: 3,294 + query_images_2: 3,058 + query_images_3: 5,451 + query_images_4: 6,556), which is slightly less than the 18,361 images stated in the paper. Could the discrepancy be due to an issue with the compressed file, or might it be related to a problem with my decompression process?
